GS_example.py

```diff
@@ -17,9 +17,13 @@
 if __name__ == "__main__":
     params = PropagationParams.get_example_propagation_data()
 
+    # Choose proper propagation parameters
     params.sigma = 4
```

Comment:
What is `sigma` exactly? You have a phd but I dont so its not so straightforward for me :D